src/Uno.UI/Extensions/ViewHelper.Android.cs

```diff
@@ -168,6 +168,32 @@ public static float LogicalToPhysicalFontPixels(float value)
 			return (int)((value * FontScale) + .5f);
 		}
 
+		/// <summary>
+		/// Gets the physical representation of the provided logical pixels.
+		/// </summary>
+		/// <param name="value">The logical value</param>
+		/// <returns>The pixels value</returns>
+		[MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
what is the difference between this one and LogicalToPhysicalPixels ? Should it be noted in the summary/remarks?

Reply:
Right, updated!